statistics/column.go

```diff
@@ -462,3 +462,18 @@ func (s StatsLoadedStatus) StatusToString() string {
 	}
 	return "unknown"
 }
+
+// IsAnalyzed indicates whether the column is analyzed.
+func (c *Column) IsAnalyzed() bool {
+	return c.StatsVer != Version0
+}
+
+// StatsCollected indicates whether the column stats are collected.
+// Note: the function merely talks about whether the stats are collected, regardless of the stats loaded status.
+// The function is used to decide StatsLoadedStatus.statsInitialized when reading the column stats from storage.
+func (c *Column) StatsCollected() bool {
```

Comment:
I think we need a comment to explain its difference from `IsAnalyzed()`.
And maybe `StatsAvailable` is a possible better name?

Reply:
Updated.